src/modules/heritability_proxy/run-gls-base.sh

```diff
@@ -9,7 +9,7 @@ fi
 echo "Env: ${ENV}"
 source ../../../env/${ENV}/global.env
 
-IMAGE_URI=gcr.io/${GOOGLE_CLOUD_PROJECT_ID}/heritability:v0.02-debug
+IMAGE_URI=gcr.io/${GOOGLE_CLOUD_PROJECT_ID}/heritability:v0.03-debug
```

Comment:
similar to previous note -- can we avoid hardcoding version numbers here and load them from the global.env?

Reply:
Fixed in #394.